main.go

```diff
@@ -190,13 +191,17 @@ func (m *mkcert) Run(args []string) {
 		if ip := net.ParseIP(name); ip != nil {
 			continue
 		}
+		if email, err := mail.ParseAddress(name); err == nil {
+			args[i] = email.Address
```

Comment:
Why modify the argument here?

Reply:
Otherwise, someone could pass `Foo Bar <foo@bar.com>` in and we'd try to put that in the SAN. I guess an alternative would be to check that `email.Address` matches the `name`.